### PR TITLE
Guacamole Service properties documentation

### PR DESCRIPTION
**Pull request by oliver7598**
# PR for issue
Partially completes #1376 

## What is being addressed
Additional documentation to highlight the exposed properties when creating a Guacamole workspace service, their default values and a description of their effect.

### CODE REVIEW

File: docs/tre-templates/workspace-services/guacamole.md [hunk 1]
```diff
@@ -33,3 +33,16 @@ Service Tags:
   make bundle-build DIR=./templates/workspace_services/guacamole
   make bundle-install DIR=./templates/workspace_services/guacamole
   ```
+
+## Guacamole Workspace Service Configuration
+
+On deploying a Guacamole service into a workspace the following properties are exposed.
```
Comment (eladiw) [inline docs/tre-templates/workspace-services/guacamole.md:39]:
```suggestion
When deploying a Guacamole service into a workspace the following properties need to be configured.
```

**Review by eladiw — APPROVED** [review]

File: docs/tre-templates/workspace-services/guacamole.md (hunk 1 above)
Comment (eladiw) [reply docs/tre-templates/workspace-services/guacamole.md:39]: maybe mention that the last property is mandatory and the rest are optional?
you can list the mandatory property first.

Comment (oliver7598) [reply docs/tre-templates/workspace-services/guacamole.md]: @eladiw good point.  Will need doing separate to this PR but currently the Client ID is not set as a required property in the template_schema.json file for the Guacamole Service despite having no default.

Comment (eladiw) [reply docs/tre-templates/workspace-services/guacamole.md]: Even if its not marked as mandatory (wrongfully) I guess the doc can still be correct. wdyt

Comment (oliver7598) [reply docs/tre-templates/workspace-services/guacamole.md]: Agree. Whether the schema highlights it or not it is still mandatory to successfully deploy the workspace service

Comment (oliver7598) [reply docs/tre-templates/workspace-services/guacamole.md]: All suggested changes implemented

**Review by eladiw — APPROVED** [review]

**Pull request merged**